modules/github-repository/repository.tf

```diff
@@ -1,4 +1,5 @@
 resource "github_repository" "repository" {
+  count                  = var.existing_repository == null ? 1 : 0
   name                   = var.name
```

Comment:
please have new line before/after "count" in all module/resource/data blocks to just separate list of module own variables, this a bit improves readability of code